docs/ldml/tr35-keyboards.md

```diff
@@ -1321,7 +1322,9 @@ _Attribute:_ `modifiers` (required for `hardware`)
 >
 > For hardware layouts, the use of `@modifiers` as an identifier for a layer is sufficient since it is always unique among the set of `layer` elements in each  `form`.
 >
-> The set of modifiers must match `(none|([A-Za-z0-9]+)( [A-Za-z0-9]+)*)`
+> This attribute is a list of lists. It is a comma-separated (`,`) list of modifier sets, and each modifier set is a space-separated list of modifier components.
+>
+> The set of modifiers must match `(none|([A-Za-z0-9]+)( [A-Za-z0-9]+)*)(,[ ]?([A-Za-z0-9]+)( [A-Za-z0-9]+)*)*`
```

Comment:
Thoughts:
* This regex prevents extra whitespace between modifiers. Is that intended?
* This regex is hard enough to read that I wonder if it is helpful to include in the spec?
* Apart from `_`, the character classes here are just `\w`. Could we go with:

`(none|\w+([ ]+\w+)*(,[ ]*\w+([ ]+\w+)*)*`

Reply:
I find it is simpler to say and easier to understand as:  

Each modifier in each set must match `[A-Za-z0-9]+`

I assume that extra whitespace is ok, eg ctrl, crtl  shift,shift

Reply:
Also should be "This attribute is either `none` or a list of lists.. " and add that extra whitespace is not significant.